### ISSUE TITLE
Installation Error with latest StyleCop Extension (2019.3.0)

### ISSUE THREAD

**Issue by softworkz** [StyleCop/StyleCop.ReSharper#32]
![image](https://user-images.githubusercontent.com/4985349/70769255-1340b200-1d69-11ea-951d-f5dde5b889eb.png)

Installation errors shows:

```
Completed with 5 errors.
05:20:10.659 |E| Error resolving type StyleCop.Rule from assembly “StyleCop, Version=6.1.0.0, Culture=neutral, PublicKeyToken=f904653c63bc2738”, request originating from assembly “ReSharperPlugin.StyleCop, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null”. Could not find a referenced assembly by the TT_ASSEMBLYREF token.
05:20:10.735 |E| Error resolving type StyleCop.Violation from assembly “StyleCop, Version=6.1.0.0, Culture=neutral, PublicKeyToken=f904653c63bc2738”, request originating from assembly “ReSharperPlugin.StyleCop, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null”. Could not find a referenced assembly by the TT_ASSEMBLYREF token.
05:20:10.741 |E| Error resolving type StyleCop.StyleCopCore from assembly “StyleCop, Version=6.1.0.0, Culture=neutral, PublicKeyToken=f904653c63bc2738”, request originating from assembly “ReSharperPlugin.StyleCop, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null”. Could not find a referenced assembly by the TT_ASSEMBLYREF token.
05:20:10.743 |E| Error resolving type StyleCop.StyleCopCore from assembly “StyleCop, Version=6.1.0.0, Culture=neutral, PublicKeyToken=f904653c63bc2738”, request originating from assembly “ReSharperPlugin.StyleCop, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null”. Could not find a referenced assembly by the TT_ASSEMBLYREF token.
05:20:10.744 |E| Error resolving type StyleCop.Rule from assembly “StyleCop, Version=6.1.0.0, Culture=neutral, PublicKeyToken=f904653c63bc2738”, request originating from assembly “ReSharperPlugin.StyleCop, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null”. Could not find a referenced assembly by the TT_ASSEMBLYREF token.

```

**Comment by softworkz** [comment]
It still gets installed in some way, but it doesn't look right:

![image](https://user-images.githubusercontent.com/4985349/70769382-982bcb80-1d69-11ea-8ad2-8875ea645d91.png)

**Comment by NikolayIT** [comment]
Same here:

![image](https://user-images.githubusercontent.com/3106986/70787182-d4295580-1d96-11ea-8eee-0601030a1195.png)

**Comment by supershowwei** [comment]
I have same issue.

```
Completed with 5 errors.
11:00:54.932 |E| Error resolving type StyleCop.Rule from assembly “StyleCop, Version=6.1.0.0, Culture=neutral, PublicKeyToken=f904653c63bc2738”, request originating from assembly “ReSharperPlugin.StyleCop, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null”. Could not find a referenced assembly by the TT_ASSEMBLYREF token.
11:00:55.018 |E| Error resolving type StyleCop.Violation from assembly “StyleCop, Version=6.1.0.0, Culture=neutral, PublicKeyToken=f904653c63bc2738”, request originating from assembly “ReSharperPlugin.StyleCop, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null”. Could not find a referenced assembly by the TT_ASSEMBLYREF token.
11:00:55.020 |E| Error resolving type StyleCop.StyleCopCore from assembly “StyleCop, Version=6.1.0.0, Culture=neutral, PublicKeyToken=f904653c63bc2738”, request originating from assembly “ReSharperPlugin.StyleCop, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null”. Could not find a referenced assembly by the TT_ASSEMBLYREF token.
11:00:55.022 |E| Error resolving type StyleCop.StyleCopCore from assembly “StyleCop, Version=6.1.0.0, Culture=neutral, PublicKeyToken=f904653c63bc2738”, request originating from assembly “ReSharperPlugin.StyleCop, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null”. Could not find a referenced assembly by the TT_ASSEMBLYREF token.
11:00:55.024 |E| Error resolving type StyleCop.Rule from assembly “StyleCop, Version=6.1.0.0, Culture=neutral, PublicKeyToken=f904653c63bc2738”, request originating from assembly “ReSharperPlugin.StyleCop, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null”. Could not find a referenced assembly by the TT_ASSEMBLYREF token.

Microsoft Windows NT 10.0.18363.0 (Windows 10 Pro, version 6.3, build 18363, release 1909)
Microsoft .NET Framework 4.8.03752 (4.8 Release 528040)
Total memory. Virtual = 127 TB; Physical = 15.9 GB; Page size = 30.8 GB
Used Managed = 21.7 MB. Available Virtual = 127 TB; Physical = 4.72 GB; Page size = 6.28 GB
Time zone: (UTC+08:00) 台北
Input languages: zh-TW (中文 (繁體) - 美式鍵盤). Default: zh-TW (中文 (繁體) - 美式鍵盤)
Visual Studios
  10.0: Visual Studio 2010, Version 10.0.40219.1
  12.0: Visual Studio 2013, Version 12.0.40629.0
  15.0_ab2056ee: Visual Studio Enterprise 2017, Version 15.9.28307.960 (Nickname="" Preview=False Incomplete=False)
  16.0_21bc1a45: Visual Studio Enterprise 2019, Version 16.4.29613.14 (Nickname="" Preview=False Incomplete=False)
Installer
  Wave: 193
  Identity: 193.0.20191210.134354
  Gallery: Local disk package gallery
  Per-machine: False
  Run as administrator: True
  Raw command line: /HostsToReInstall=ReSharperPlatformVs12
  Parsed options: /HostsToReInstall=ReSharperPlatformVs12
Products root: C:\Users\***\AppData\Local\JetBrains (free 72074 MB of 274115 MB)
Products before install
  dotPeek193 (Wave=193)
    dotPeek 2019.3 (193.0.20191207.71320)
  dotTrace193 (Wave=193)
    dotTrace 2019.3 (193.0.20191207.71700)
  ReSharperPlatformVs12 (Wave=193 VS=12.0)
    ReSharper 2019.3 (193.0.20191207.71320)
    dotTrace (193.0.20191207.71700)
  ReSharperPlatformVs15_ab2056ee (Wave=193 VS=15.0 Id=ab2056ee)
    ReSharper 2019.3 (193.0.20191207.71320)
    dotTrace (193.0.20191207.71700)
  ReSharperPlatformVs16_21bc1a45 (Wave=193 VS=16.0 Id=21bc1a45)
    ReSharper 2019.3 (193.0.20191207.71320)
    dotTrace (193.0.20191207.71700)
Install targets
  update  ReSharperPlatformVs12 at C:\Users\***\AppData\Local\JetBrains\Installations\ReSharperPlatformVs12_001
Products after install
  dotPeek193 (Wave=193)
    dotPeek 2019.3 (193.0.20191207.71320)
  dotTrace193 (Wave=193)
    dotTrace 2019.3 (193.0.20191207.71700)
  ReSharperPlatformVs12 (Wave=193 VS=12.0)
    ReSharper 2019.3 (193.0.20191207.71320)
    dotTrace (193.0.20191207.71700)
    StyleCop StyleCop 2019.3.0
  ReSharperPlatformVs15_ab2056ee (Wave=193 VS=15.0 Id=ab2056ee)
    ReSharper 2019.3 (193.0.20191207.71320)
    dotTrace (193.0.20191207.71700)
  ReSharperPlatformVs16_21bc1a45 (Wave=193 VS=16.0 Id=21bc1a45)
    ReSharper 2019.3 (193.0.20191207.71320)
    dotTrace (193.0.20191207.71700)

```

**Comment by supershowwei** [comment]
Hey guys, it is fixed in 2019.3.1

**Comment by matkoch** [comment]
Thanks for confirming.
